datasets.py

```diff
@@ -0,0 +1,64 @@
+from random import Random
+import ipaddr
+import datetime as dt
+
+from interface import Entry
+
+LOWEST_IP = 2 ** 30  # 64.0.0.0
+HIGHEST_IP = 2 ** 31  # 128.0.0.0
+LOWEST_TIME = dt.datetime(2001, 1, 1).toordinal()
+HIGHEST_TIME = dt.datetime(2016, 1, 1).toordinal()
+PROTOCOLS = ['TCP', 'UDP', 'DNS', 'HTTP', 'SSH', 'FTP']
+
+rnd = Random(0)
```

Comment:
No need to generate seed..
you can just use `import random` and then `random.randint` and the others..

Reply:
Not my code - from the exercise documentation